js/repl/loadBabel.js

```diff
@@ -0,0 +1,43 @@
+import loadBuildArtifacts from "./loadBuildArtifacts";
+import loadScript from "./loadScript";
+import { BabelState, LoadScriptCallback } from "./types";
+
+export default function loadBabel(config: BabelState, cb: LoadScriptCallback) {
+  function doLoad(url) {
+    loadScript(url, cb);
+  }
+
+  // See if a CircleCI build number was passed in the path
+  // Prod (with URL rewriting): /repl/build/12345/
+  // Dev: /repl/#?build=12345
```

Comment:
Heh, this is my comment (this is copied across from `7.html`). I try to keep them informative. 😛